### ISSUE TITLE
event examples include spurious 'age' field

### ISSUE THREAD

**Issue by richvdh** [matrix-org/matrix-doc#1524]
most of the event examples include an `age` field, which isn't really part of the event - in particular it will be absent over federation. `/sync` and `/messages` will add an `age`, but put it in the `unsigned` object. `/intialSync` probably *does* put an `age` where the examples suggest, but it's deprecated.

The example for [m.room.redaction](https://matrix.org/docs/spec/client_server/unstable.html#m-room-redaction) puts `age` under `unsigned`, but I'm not sure that's very clear either.

Basically I think we should remove both `age` and `unsigned` from these examples.

**Comment by richvdh** [comment]
related: https://github.com/matrix-org/matrix-doc/issues/684
